golang/cosmos/x/swingset/types/default-params.go

```diff
@@ -100,6 +103,12 @@ var (
 		// UintMapEntry{VatCleanupSnapshots, DefaultVatCleanupSnapshots},
 		// UintMapEntry{VatCleanupTranscripts, DefaultVatCleanupTranscripts},
 	}
+
+	// DefaultBundleUncompressedSizeLimitBytes is the (exclusive) limit on uncompressed bundle size.
+	// We must ensure there is an exclusive int64 limit in order to detect an underflow.
+	// Bundles must be strictly less than this size.
+	DefaultBundleUncompressedSizeLimitBytes int64 = 10 * 1024 * 1024 // 10MB
+	DefaultChunkSizeLimitBytes              int64 = 512 * 1024       // 512KB
```

Comment:
In my patch to `@endo/bundle-source` I computed the maximum byte limit as:

```js
/**
 * CometBFT RPC Node default max_body_bytes is 1,000,000 , and we hex encode the
 * bundle before POSTing (2 hex chars per input byte), with an ample size
 * reserved for overhead (10,000 bytes).
 */
export const AGORIC_MAX_BYTE_LIMIT = (1_000_000 / 2) - 10_000;
```

I strongly suggest changing this default to be:

```suggestion
	// CometBFT RPC Node default max_body_bytes is 1,000,000 , and we hex encode the
	// bundle before POSTing (2 hex chars per input byte), with an ample size
	// reserved for overhead (10,000 bytes).
	DefaultChunkSizeLimitBytes              int64 = 1e6 / 2 - 10e3 // 490kB
```